media/static-bundles.json

```diff
@@ -1031,17 +1031,6 @@
       ],
       "name": "firefox_whatsnew_67"
     },
-    {
-      "files": [
-        "js/base/uitour-lib.js",
-        "js/base/mozilla-fxa.js",
-        "js/base/mozilla-fxa-init.js",
-        "js/base/mozilla-fxa-form.js",
-        "js/base/mozilla-monitor-button.js",
-        "protocol/js/protocol-modal.js"
-      ],
-      "name": "firefox_whatsnew_67.0.5"
```

Comment:
This was also accidentally removed but a rebase should put it back.

Reply:
this should be fixed now